images/autoowners/build_and_push_from_local.sh

```diff
@@ -12,19 +23,22 @@ WORK_DIR_PARENT="/tmp/github.com/openshift"
 mkdir -p "$WORK_DIR_PARENT"
 cd "$WORK_DIR_PARENT"
 WORK_DIR="$WORK_DIR_PARENT/ci-tools"
-[ ! -d "$WORK_DIR" ] && git clone https://github.com/openshift/ci-tools.git
+[ ! -d "$WORK_DIR" ] && git clone https://github.com/dhiller/ci-tools.git
 
 cd "$WORK_DIR"
+git checkout add-signoff-to-autoowners-2
+git pull
 export GOPROXY=off
 export GOFLAGS=-mod=vendor
 make install
 
-IMAGE_NAME="kubevirtci/autoowners"
+IMAGE_NAME="quay.io/kubevirtci/autoowners"
```

Comment:
I didn't know about the kubevirtci org in quay and have starting moving some of the existing docker.io/kubevirtci images  to quay.io/kubevirt instead, do you think we should keep the two different quay orgs?

Reply:
Yes, I think the separation between kubevirt and kubevirtci makes sense. My take on this is: everything related to a kubevirt product release would belong to quay.io/kubevirt and everything related to kubevirt prow build infrastructure (which is also used by other projects) would belong to quay.io/kubevirtci.

Reply:
TBH I'm not exactly sure how strict we have been until now wrt this separation :-/